crypto/crypto.go

```diff
@@ -13,17 +13,29 @@ package crypto
 import (
 	"crypto/ed25519"
 	"crypto/rand"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
```

Comment:
discussion:

I think I was using `	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"`

Any differences you know between them?